web-app/js/portal/cart/DownloadHandler.js

```diff
@@ -89,7 +89,8 @@ Portal.cart.DownloadHandler._downloadHandlerConstructorForProtocol = function(pr
             Portal.cart.PythonDownloadHandler
         ],
         'IMOS:AGGREGATION--bodaac': Portal.cart.BodaacDownloadHandler,
-        'IMOS:AGGREGATION--gogoduck': Portal.cart.GogoduckV1DownloadHandler
+        'IMOS:AGGREGATION--gogoduck': Portal.cart.GogoduckV1DownloadHandler,
+        'OGC:WPS--gogoduck': Portal.cart.GogoduckDownloadHandler
```

Comment:
I think this is what we agreed on for the protocol. For the NetCDF Subset Service the protocol will be `OGC:WPS--netcdf-subset-service`.